topics/epigenetics/metadata.yaml

```diff
@@ -54,6 +54,25 @@ material:
     contributors:
       - joachimwolff
       - dpryan79
+  -
+    title: "EWAS data analysis"
+    type: "tutorial"
+    name: "ewas-suite"
+    zenodo_link: "https://zenodo.org/record/1251211#.WwREQ1Mvz-Y"
+    galaxy_tour: yes
+    hands_on: yes
+    slides: yes
+    workflows: yes
+    questions:
+      - "What is methylation and why it cannot be recognised by a normal NGS procedure?"
```

Comment:
.... by normal DNA sequencing?

Reply:
Hi Bjorn,  I didn't save a changes when I was editing copied example. 
Now I've provide my questions and objectives, thanks for spotting this.